tests/test_data.py

```diff
@@ -184,6 +184,73 @@ def create_mock_items_presigned(stac_id: str = DUMMY_STAC_IDS[0], polarization:
     },
 }
 
+REPEAT_REQUEST = {
```

Comment:
in case that wasn't done prior: Are you able to edit the properties a bit - thanks

Reply:
Good call, thanks. Pulled out the IDs.